app/views/teacher_training_adviser/steps/_qualification_required.html.erb

```diff
@@ -1,10 +1,20 @@
 <% @page_title = "Get the right GCSEs or equivalent qualifications" %>
 
-<h1 class="govuk-heading-l">Get the right GCSEs or equivalent qualifications</h1>
+<h1 class="govuk-heading-l">We're sorry, but you need the right GCSEs to be eligible for this service</h1>
 <div class="govuk-form-group">
-    <p>You need to pass the required GCSEs or complete the equivalency test in order to train to teach in England.</p>
-    <p>To get the right GCSEs you will need to find a <a href="https://nationalcareers.service.gov.uk/find-a-course/search">GCSE training provider</a>. Make sure when you visit this website, you search for GCSE courses.</p>
-    <p>Once you find a training provider they may ask you to sit a GCSE, equivalency test, or offer other evidence to demonstrate your ability.</p>
-    <p>Or</p>
-    <p>Return to the <%= link_to_git_site %> homepage.</p>
+    <p>To train to be a teacher, teacher training providers ask you to show your ability through GCSEs. You'll need the following GCSEs (or an equivalent):</p>
+
+    <ul>
+      <li>grade 4 (C) in maths</li>
+      <li>grade 4 (C) in English</li>
+      <li>grade 4 (C) in science, if you want to teach primary</li>
+    </ul>
+
+    <p>If you do not have the right GCSEs, you can get them through a <a href="https://nationalcareers.service.gov.uk/find-a-course/search">GCSE training provider</a>. Make sure when you visit the website, you search for GCSE courses.</p>
+
+    <p>Or, you can take an equivalency test, or offer other evidence to demonstrate your ability in these subjects.</p>
+
+    <p><%= link_to_git_site("Find out more about the qualifications you need to teach", "is-teaching-right-for-me/qualifications-you-need-to-teach", target: "_blank") %>.</p>
+
+    <p>You can also find out more about teaching on the <%= link_to_git_site %> website.</p>
```

Comment:
Should 'find out more about teaching on the GIT website' be the link - better for accessibility to give more context in the link text?